Lesson17/serkan/script.js

```diff
@@ -62,10 +84,10 @@ console.log('- - - - - - - - - - - - - - - - - ');
 
 // 10. Write a for...in loop that prints out the keys and values of an object.
 const person = { name: "John", surname: "Doe", age: 30, city: "New York" };
- for ( const key in person){
-    console.log(`${key}: ${person[key]}`);
- }
- 
+for ( const key in person){
+  console.log(`${key}: ${person[key]}`);
+}
+
 
 // 11. (H) Write a for...in loop that sums the values of an object.
-const obj = { a: 10, b: 20, c: 30, d: 23 };
\ No newline at end of file
+const obj = { a: 10, b: 20, c: 30, d: 23 };
```

Comment:
Solution to this exercise is missing. Try to solve it similarly to ex. where you needed to find min/max values of array: create a variable where you'll store the sum. Update that variable while iteration though values of the object.